src/proj.rs

```diff
@@ -159,9 +169,64 @@ impl Proj {
         })
     }
 
-    /// Add a [resource file search path](https://proj.org/resource_files.html), maintaining existing entries.
+    /// Check whether network access for [resource file download](https://proj.org/resource_files.html#where-are-proj-resource-files-looked-for) is currently enabled or disabled.
     ///
-    /// Changes to the search path [_should be_](https://github.com/OSGeo/PROJ/issues/2266) reflected in all existing and subsequently-created `Proj` instances
+    /// # Safety
+    /// This method contains unsafe code.
+    fn network_enabled(&self) -> bool {
+        let res = unsafe { proj_context_is_network_enabled(self.ctx()) };
+        match res {
+            1 => true,
+            _ => false,
+        }
+    }
+
+    /// Get the URL endpoint to query for remote grids
+    ///
+    /// # Safety
+    /// This method contains unsafe code.
+    fn get_url_endpoint(&self) -> Result<String, ProjError> {
+        unsafe { _string(proj_context_get_url_endpoint(self.ctx())) }
+    }
+}
+
+impl Info for TransformBuilder {
+    #[doc(hidden)]
+    fn ctx(&self) -> *mut PJ_CONTEXT {
+        self.ctx
+    }
+}
+
+impl TransformBuilder {
```

Comment:
Naming nit, but maybe this would be clearer as `ProjBuilder` rather than introducing the concept of a "Transform" as something which might be distinct from "Proj"

Take it or leave it though - it's also fine as is.